src/adapter/lambda-edge/conninfo.ts

```diff
@@ -0,0 +1,15 @@
+import type { GetConnInfo } from '../../helper/conninfo'
+import type { CloudFrontEdgeEvent } from './handler'
+import type { Context } from '../../context'
+
+type Env = {
+  Bindings: {
+    event: CloudFrontEdgeEvent
+  }
+}
+
+export const getConnInfo: GetConnInfo = (c: Context<Env>) => ({
+  remote: {
+    address: c.env.event.Records[0].cf.request.clientIp,
```

Comment:
Can you add an `addressType` property?

Reply:
I've added it.

Reply:
Oops,  if it is "uknown", we must not set `unknown`(https://github.com/honojs/hono/pull/2958) since this PR will be merged into the `next`. Please remove it. Sorry for taking your time.

Reply:
Yes, if it is merged into the next branch, addressType should be removed.

Reply:
Thanks!